Testing section (#1633)
temporalio/documentation: docs/php/testing.md

Instead, we can use a light-weight test server.

> The code in `bootstrap.php` will start/stop (and download if it doesn't exist) Temporal test

Reviewer: ```suggestion
The code in `bootstrap.php` will start and stop (and download if it doesn't exist) the Temporal test server and RoadRunner for every phpunit run.
```

> server and RoadRunner for every phpunit run. test server runs as a regular server on 7233 port.

Reviewer: ```suggestion
The test server runs as a regular server on 7233 port.
```
